### DISCUSSION TITLE
Purchase Orders in Assets Import

### DISCUSSION THREAD

**Discussion by jfbarthe-sls** [snipe/snipe-it#9925] [Q&A]
Reading the docs for [importing assets](https://snipe-it.readme.io/docs/importing), I cannot find the name I need to use for Purchase Orders column in the first row of the CSV file. 
I don't believe this is a custom field. 
I hope it is an importable field as we would like to bulk import a few assets.
Any hint appreciated.
Thanks

**Comment by PlaneNuts** [comment]
I'm not sure if this is an importable field, but it looks like the column in the assets table is called order_number

**Reply by jfbarthe-sls** [reply]
It is! I figured it out. After some trials and errors, the column must be called `Order Number`.
